src/callbacks_step/analysis_dg1d.jl

```diff
@@ -120,51 +120,52 @@ function calc_error_norms(func, u, t, analyzer,
 end
 
 function integrate_via_indices(func::Func, u,
-                               mesh::StructuredMesh{1}, equations, dg::DGSEM, cache,
+                               mesh::TreeMesh{1}, equations, dg::DGSEM, cache,
```

Comment:
I exchanged the order of the functions, to have `TreeMesh` on top as for the other dimensions.